.tekton/operator-bundle-pipeline.yaml

```
@@ -105,64 +105,114 @@ spec:
     description: This sets the expiration time for intermediate OCI artifacts produced and used during builds after which they can be garbage collected.
     name: oci-artifact-expires-after
     type: string
+
   - name: operator-image-repo
```

Suggestion: how about if we call these
```suggestion
  - name: operator-image-build-repo
```
to differentiate from the second ones? `-build` means that these are repos where images get originally built to.

Similarly, `description` now needs a bit of tweaking because generic
```yaml
description: Image repository for the StackRox Operator.
```
does not make things very clear.

---

Descriptions are updated in 6466c0f4ccb366aee12e3cd3e2b2d51b296f145c and 6d4c16bf99ea6aa02b7325d8aea088c3fcb0b43d.